### PR TITLE
feat(exchange.disclaimer): not editable editor

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-41448
| License          | BSD 3-Clause

## Description

Remove tabindex attribute from bootstrap modal as it clashes with ckeditor modal

### CODE REVIEW

**Review by varun257 — COMMENTED** [review]
There are 2 modals here, bootstrap modal and a ckeditor modal created fetched dynamically.
The tabindex attribute of the ckeditor is clashing with the bootstrap modal tabindex
Hence i am removing this manually after modal is loaded.

File: packages/manager/modules/exchange/src/disclaimer/update/disclaimer-update.controller.js [hunk 1]
```diff
@@ -25,6 +25,9 @@ export default class ExchangeUpdateDisclaimerCtrl {
   loadOptions() {
     this.loadingData = true;
 
+    // Remove tabindex from modal as it clashes with ckeditor modal
+    document.getElementsByClassName('modal')[0].removeAttribute('tabindex');
```
Comment (frenautvh) [inline packages/manager/modules/exchange/src/disclaimer/update/disclaimer-update.controller.js:29]: it's kind of a dirty fix, .modal class refer to which element, the ckeditor one or the bootstrap one ?
maybe you can scope the query by className a little more narrower, i didn't check but something like :
```js
document.getElementsByClassName('.ckeditor-modal > .modal'); // just an example
```
also please use angularJS api to query elements : https://docs.angularjs.org/api/ng/service/$document

**Review by frenautvh — CHANGES_REQUESTED** [review]
see comments

File: packages/manager/modules/exchange/src/disclaimer/update/disclaimer-update.controller.js (hunk 1 above)
Comment (varun257) [reply packages/manager/modules/exchange/src/disclaimer/update/disclaimer-update.controller.js]: I cannot find a better fix for this. I cannot add class as the bootstrap modal is outside the component.
I have used angularJS api to query the modal

Comment (varun257) [reply packages/manager/modules/exchange/src/disclaimer/update/disclaimer-update.controller.js]: @frenautvh is this okay ?

**Review by frenautvh — APPROVED** [review]

**Review by ganeshkumar1989 — APPROVED** [review]

**Pull request merged**